vms/avm/vm.go

```diff
@@ -162,7 +152,7 @@ func (vm *VM) Initialize(
 	genesisBytes []byte,
 	_ []byte,
 	configBytes []byte,
-	toEngine chan<- common.Message,
+	_ chan<- common.Message,
```

Comment:
if we decide to remove this `toEngine` channel, should we also remove `toEnginee` as part of `func (vm *VM) Linearize(_ context.Context, stopVertexID ids.ID, toEngine chan<- common.Message) error`

Reply:
No, the `toEngine` channel in `Linearize` is used to communicate to the `Snowman` consensus engine.